Client-Setup-Windows.md

```diff
@@ -48,74 +50,77 @@ After having connected to retail at least once since installing Final Fantasy XI
 
 6. Once everything is done, click on "OK" then "Exit Viewer" and "Yes".
 
-# 4. Building xiloader
-
-To connect to a private server, you need to be able to direct _where_ your local client communicates. This is done through programs like [xiloader](https://github.com/zircon-tpl/xiloader/releases).
-
-**Note: Ashita has a loader built into it. Users who plan to use Ashita may skip to [Section 6 - Launcher Configuration](Client-installation-setup-Windows-%28second-part%29/#6-launcher-configuration). However, we _highly recommend_ connecting to your server for the first time with only xiloader so that you can minimize potential setup problems.**
-
-To build xiloader from [source](https://github.com/LandSandBoat/xiloader/):
-
-1. Download and install [MSVC 2015 Runtimes](https://www.microsoft.com/en-ca/download/details.aspx?id=48145) (if needed).
+# 4. xiloader
 
-2. Right click wherever you want to download the xiloader repository > Git Clone... > URL: https://github.com/zircon-tpl/xiloader.git > OK > then Close when it's done.
+To connect to a private server you need to point your client at that private server. This is done through `bootloaders` like `xiloader`.
 
-3. Execute Visual Studio Community 2019: Open a project or solution (on the right) > select the "xiloader.sln" file from the \xiloader\ folder > Open (click "OK" when asked to "Retarget Projects").
+* [Download pre-built xiloader](https://github.com/LandSandBoat/xiloader/releases)
+* [Download source code for xiloader to build for yourself](https://github.com/LandSandBoat/xiloader)
 
-4. Once it's loaded > Build > Build Solution.
-
-5. To confirm that everything was built properly, you should see:
-
-> ========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
-
-in the output console at the bottom and the xiloader executable file should be present in the \xiloader\Debug\ folder.
-Put it wherever you want but remember its path.
+**Note: Ashita comes bundled with it's own bootloader. Users who plan to use Ashita may skip to [Section 6 - Launcher Configuration](#6-launcher-configuration). However, we _highly recommend_ connecting to your server for the first time with only xiloader so that you can minimize potential setup problems.**
 
```

Comment:
Sooner rather than later, the new login server will go live, and with it, the new xiloader. Unless at0mos is going to update it, this section will have to be revised